pkg/util/protoutil/jsonpb_marshal.go

```diff
@@ -133,3 +133,8 @@ func (j *JSONPb) NewEncoder(w io.Writer) gwruntime.Encoder {
 		return errors.Errorf("unexpected type %T does not implement %s", v, typeProtoMessage)
 	})
 }
+
+// Delimiter for newline encoded JSON streams.
```

Comment:
This comment should follow the convention in this file: "Delimiter implements gwruntime.Delimited" (or whatever the name of the interface is. We should also have a static assertion right above this to ensure that the interface is indeed implemented. There's a similar assertion at the top of this file: https://github.com/cockroachdb/cockroach/blob/edc971821297410d265db694b825162403cf5c35/pkg/util/protoutil/jsonpb_marshal.go#L30.

Please make the same changes to the protobuf marshaler as well.

Reply:
The static assertion won't work unless we actually bump the grpc-gateway version, unless you want me to assert `interface{ Delimiter() []byte }`. I've updated the comments.